app/assets/stylesheets/application.scss

```diff
@@ -121,8 +133,12 @@ p.text {
   background: url("http://subtlepatterns2015.subtlepatterns.netdna-cdn.com/patterns/noisy_net.png") !important;
```

Comment:
@samehta https://github.com/samehta/fantasy-metrix/pull/9/files#r47818752 applies to all images hosted on other sites that are used by fantasy-metrix.